shift_detector/precalculations/lda_embedding.py

```diff
@@ -19,55 +20,64 @@ def __init__(self, cols, n_topics=20, n_iter=10, lib='sklearn', random_state=0,
         self.model = None
         self.trained_model = None
         self.cols = None
-        if n_topics < 2:
-            raise ValueError('Number of topics has to be at least 2')
+        self.stop_words = stop_words
+        self.max_features = max_features
+
         if n_topics != 'auto':
```

Comment:
Shouldn't is be `n_topics == 'auto'`? Then you check whether `n_topics` is an `int`. And then if `n_topics` is smaller than 2?